ext/opentelemetry-ext-django/README.rst

```diff
@@ -0,0 +1,15 @@
+OpenTelemetry Django Instrumentation
+====================================
+
+This instrumentation creates OpenTelemetry spans for Django applications.
+
+Once this package is installed, run your Django application like this:
+
+`opentelemetry-python-autoinstrumentation python3 manage.py runserver`
```

Comment:
I think the README should follow a similar format to the other [READMEs](https://github.com/open-telemetry/opentelemetry-python/blob/master/ext/opentelemetry-ext-flask/README.rst) where it just shows you what package to install.

Reply:
Fixed